test/elastic_record/index/documents_test.rb

```diff
@@ -99,7 +99,25 @@ def test_expired_scroll_error
     )
 
     scroll_enumerator.request_more_hits
-    index.connection.json_delete '/_search/scroll', scroll_id: scroll_enumerator.scroll_id
+    index.delete_scroll(scroll_enumerator.scroll_id)
+    assert_raises ElasticRecord::ExpiredScrollError do
+      scroll_enumerator.request_more_hits
+    end
+  end
```

Comment:
The above several lines seem out of place. Are you "borrowing" the state of the test to test that delete_scroll works?

Reply:
Without using webmock or some other mock/spy it's difficult to assert that the context is closed (the state of elastic search stats request doesn't update quickly enough for the test), so this assertion/extra request is necessary

Reply:
Yeah but there’s an explicit call to delete the scroll id

Reply:
Yeah but there’s an explicit call to delete the scroll id